swagger.yml

```diff
@@ -3556,42 +3608,11 @@ paths:
                     properties:
                       technologies:
                         type: array
+                        description: All available technologies must be returned by default with values at zero if not used.
                         items:
-                          type: object
-                          properties:
-                            product:
-                              type: string
-                              description: technologial product code
-                            Compliance:
-                              type: number
-                              description: |-
-                                a metric ranging [0..1] calculated as follows:
-
-                                if a ULA exists for a given product, its compliance := 1. When calculating the total, this product's purchasedLicenses := usedLicenses
-                                else if usedLicenses <= purchasedLicenses the product's compliance := 1. When calculating the total, this product's purchasedLicenses := usedLicenses
-                                else this product's compliance := purchasedLicenses / usedLicenses.
-                            unpaidDues:
-                              type: number
-                              description: |-
-                                "unpaidDues" = difference between calculated cost of used and purchased licenses:
-
-                                if usedLicensesCost <= purchasedLicensesCost, then unpaidDues := 0
-                                else unpaidDues := usedLicensesCost - purchasedLicensesCost
-                            hostsCount:
-                              type: number
-                              description: number of hosts found supporting the current product / total hosts found
```

Comment:
Why don't you move the description to the component?

Reply:
Done !